### ISSUE TITLE
wtfnode doesn't work if cwd is not inside wtfnode package

### ISSUE THREAD

**Issue by efokschaner** [myndzi/wtfnode#15]
Recently tried to use new version of wtfnode and ran into this issue. It seems that `proxy.js` assumes that the wtfnode `index.js` is in the cwd at https://github.com/myndzi/wtfnode/blob/cec4e3706157aeacc2a6598d534af09834d38518/proxy.js#L11

It produces the following error output:
```
$ wtfnode main.js
module.js:442
    throw err;
    ^

Error: Cannot find module 'C:\dev\node\http-echo\index.js'
    at Function.Module._resolveFilename (module.js:440:15)
    at Function.Module._load (module.js:388:25)
    at Module.runMain (module.js:575:10)
    at run (bootstrap_node.js:352:7)
    at startup (bootstrap_node.js:144:9)
    at bootstrap_node.js:467:3
```

PR with fix incoming.

**Comment by myndzi** [comment]
Ah, silly of me. Nice catch, thanks.

**Comment by myndzi** [comment]
Published 0.5.2
